kernel/kernel-util/src/main/java/org/sakaiproject/util/EmailNotification.java

```diff
@@ -645,6 +645,24 @@ protected int getOption(User user, String notificationId, String resourceFilter,
 		String type = getType(resourceFilter);
 		if (type != null)
 		{
+            // First, check the overrides.
```

Comment:
I'm using soft tabs, 4 spaces per tab, whereas the rest of the file seems to be using hard tabs. What's the policy?